### ISSUE TITLE
The URL or file path given does not exist.

### ISSUE THREAD

**Issue by bestsancho** [HydraWiki/PDFEmbed#16]
Hi,

i am running Mediawiki 1.29.1 and PDFEmbed 2.0.1. But when I try to embed a PDF-File I get "The URL or file path given does not exist." at the position where the embedded PDF should appear. 

Do you have hints to change this?

Thank you in advance.
Sancho

**Comment by guindelo** [comment]
Hi, Can you resolve the problem?

Hi have the same problem yet.

Can you help me please?

Thanks.

guindelo

**Comment by bestsancho** [comment]
Hello guindelo,

sorry, until now i didn't a solution for this problem.

cu
Sancho

**Comment by guindelo** [comment]
Hello Sancho, 
I revise antother time the problem, and now, it work's.

I do 2 things, first, Do another time de configuration/installation page of my wiki.

after the urlofyourwiki/index.php/mw-config .

The other think I change is the main url on the Localsettings.php, The last week I do an Redirection by 301, an I think this maybe the problem, in my localsettings i put the main url with www.

## The protocol and server name to use in fully-qualified URLs
$wgServer = "http://www.yourdomain.com";

I hope it works for you.

guindelo.

**Comment by bestsancho** [comment]
Hello guindelo,

thank you so much for help. But until now I didn't get a glue.

Before I installed the extension PDFEmbed I made some experiments with my url of wiki  and found out that are many difficulties with it for every single user access from their PC. It works now in the way

$wgServer = "http://<server-name>"

Every change at this position made access problems from some users' PC.

So I hope that there is a solution at configuration of extension PDFEmbed.

Sancho

**Comment by bestsancho** [comment]
$wgServer = "http://servername"

**Comment by bestsancho** [comment]
Hello guindelo,

now I have found the solution. 

Today I downloaded PDFEmbed from https://codeload.github.com/HydraWiki/PDFEmbed/zip/v2.0.1again and restored it to mediawiki's extension directory . Then I  put the line

wfLoadExtension( 'PDFEmbed' ); 

at the end of Localsettings.php.

After this I uploaded a new pdf file and my way of usage is <pdf>File:Example.pdf</pdf>. This works for me now. 

Yes, I know, it looks like the text in the documentation. But I swear that it did not work for me when I tried it many times before. Some links in google showed some variations. I tried them without any success. I also varied the position of "wfLoadExtension( 'PDFEmbed' ); " in Localsettings.php. Now it is at the very end and it works.

Thank you so much.

Sancho
